.github/workflows/node-ci.yml

```diff
@@ -239,7 +239,7 @@ jobs:
                   ssh-private-key: ${{ secrets.KHAN_ACTIONS_BOT_SSH_PRIVATE_KEY }}
             # Make sure our packages aren't growing unexpectedly
             # This must come last as it builds the old code last and so leaves the
-            # wrong code in place for the next job.
+            # wrong code in place for the next job; in other words, it leaves the repo on a base branch.
```

Comment:
Nice.  I did like how in webapp they have a comment _after_ this step saying something like `# Don't add any steps after (see comment above this last step)` to avoid someone not seeing this comment you've just edited here.

Reply:
Added! :)